angular-tour-of-heroes/src/app/heroes/heroes.component.ts

```diff
@@ -45,4 +45,15 @@ export class HeroesComponent implements OnInit {
       this.heroes.push(r);
     });
   }
+
+  deleteHero(hero: Hero): void {
```

Comment:
I like the fact I'm starting to understand this and create stuff without fully following the tutorial (finally)